src/stack_manager.py

```diff
@@ -158,20 +162,24 @@ def pair2idx(c, r):
 
         grid = [[None]*rows for _ in range(cols)]
         for i in range(len(images)):
-            c, r = scope.children[i].tag.gridposition or idx2pair(next_gridposition)  # TODO: check order of r, c
+            c, r = scope.children[i].tag.gridposition or idx2pair(
+                next_gridposition)  # TODO: check order of r, c
```

Comment:
presumably the order has been found correct, or something would be broken? I suspect we can delete the TODO at this point

Reply:
me too